apex_launchtest/examples/args.test.py

```diff
@@ -64,7 +64,7 @@ def generate_test_description(ready_fn):
 class TestTerminatingProcessStops(unittest.TestCase):
 
     def test_proc_terminates(self):
-        self.proc_info.assertWaitForShutdown(process=dut_process, timeout=10)
+        self.proc_info.assertWaitForShutdown(proc=dut_process, timeout=10)
```

Comment:
Let me clarify, in my opinion we are going from a good argument/variable name (`process`) to a less good one (`proc`). Of course that's in my opinion, due to my preference to avoid abbreviation where ever possible. It makes it easier to search for keywords and it helps people for whom English is not their first language.

There might other places where an abbreviation is being used now where it wasn't before, but that's only one I commented on, even though the same logic would apply in each of those cases.

Reply:
This is obviously subject and we're inconsistent in the ROS 2 project, because we have things like `cmd_args` or `env`. So, it's really just in this case, the change here seems like a step backward, but again it's very subjective.

Reply:
It's this way because of flake8 (indirectly).  When this was apex_rostest, the argument was 'node'.  When I changed everything to be apex_launchtest, node became proc instead of 'process' so I wouldn't have to reformat a bunch of lines that were close to the line limit.

It's good feedback about searching and ESL - I can make everything 'process' instead of 'proc'  There were other breaking changes in here, so it doesn't make it any more breaking